Swiftfin/Views/UserSignInView/UserSignInView.swift

```diff
@@ -388,4 +371,25 @@ struct UserSignInView: View {
             Text("Set pin for new user.")
         }
     }
+
+    func handleSignIn(_ event: UserSignInViewModel.Event) {
```

Comment:
Similar to https://github.com/jellyfin/Swiftfin/pull/1084#discussion_r1632113562, make `private` and move to above, probably best above all of the QuickConnect/sign in stuff.